lrrbot/__init__.py

```diff
@@ -172,7 +170,9 @@ def wrapper(function):
 	def on_connect(self, conn, event):
 		"""On connecting to the server, join our target channel"""
 		log.info("Connected to server")
-		self.twitchclient(conn, 3)
+		conn.cap("REQ", "twitch.tv/membership") # get join/part messages
```

Comment:
Yeah... there's a couple of parts of the bot that don't work without join/part messages, in particular the mode tracking... but we're doing our own tracking of who's +o for other reasons anyway, and the bot's been working fine for some time without join/part messages, so I guess it isn't really needed.